packages/vue/src/components/confirm-sign-in.vue

```diff
@@ -20,7 +20,7 @@ const facade: UseAuthenticator = useAuthenticator();
 const { submitForm, toSignIn, updateForm } = facade;
 const { user, error, isPending } = toRefs(facade);
 
-const challengeName = computed(() => user.value.challengeName);
+const challengeName = computed(() => user.value.nextStep.signInStep);
```

Comment:
Think we can avoid a breaking change here if we map `nextStep.signInStep` to `challengeName`. Also avoids exposing JS return types directly on `user`

Reply:
yep, can change that as we discussed